test/unit/constants/uuidV4Regex.js

```diff
@@ -10,9 +10,4 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-export default ({ sendFetchRequest, sendBeaconRequest }) => {
-  return ({ url, body, documentMayUnload }) => {
-    const method = documentMayUnload ? sendBeaconRequest : sendFetchRequest;
-    return method(url, body);
-  };
-};
```

Comment:
Git correlated these files, but they're not related at all. injectNetworkStragy.js has been removed. At one point it used to do more, but it wasn't providing much value anymore.